From b07f6ecd6ab83b788301a555dc546b9a5c9dd8a2 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 25 Jan 2023 19:19:34 -0800
Subject: [PATCH] Include missing <cstdint>

This error is seen with gcc-13 where include headers are not implicitly
included [1]

Fixes errors e.g.
error: 'uint32_t' does not name a type

[1] https://www.gnu.org/software/gcc/gcc-13/porting_to.html

Upstream-Status: Submitted [https://github.com/KhronosGroup/VK-GL-CTS/pull/378]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 framework/common/tcuDefs.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/framework/common/tcuDefs.hpp b/framework/common/tcuDefs.hpp
index ad3d0736d..6dfac10fb 100644
--- a/framework/common/tcuDefs.hpp
+++ b/framework/common/tcuDefs.hpp
@@ -26,6 +26,7 @@
 #include "deDefs.hpp"
 #include "qpTestLog.h"
 
+#include <cstdint>
 #include <string>
 #include <stdexcept>
 
-- 
2.39.1

